### PR TITLE
Us103 enter dob

### PR DESCRIPTION
**Pull request by Phemus**
This pull request implements the enter dob function for the non verify login. it also updates the page design to closer match the prototype direction

### CODE REVIEW

File: application/templates/enterdob.html [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+{% block content %}
+<div class="main" id="content">
+    {{ super() }}
+    <div>&nbsp;</div>
+    <h1 class="heading-small">What is your date of birth?</h1>
+    <form id="enter-dob-form" action="/searchdeed/enter-dob" method="POST">
+
+      <legend class="visuallyhidden">Date of birth</legend>
+
+{#      <div class="form-date">#}
```
Comment (tsangsimon) [inline application/templates/enterdob.html]: left some HTML comments here

File: application/deed/searchdeed/views.py [hunk 2]
```diff
@@ -10,11 +12,47 @@ def search_deed_main():
...
 
 
+def validate_dob(form):
+    error = None
+    try:
+        day = int(form["dob-day"])
+        month = int(form["dob-month"])
+        year = int(form["dob-year"])
+
+        datetime.datetime(year, month, day)
```
Comment (endamccormack) [inline application/deed/searchdeed/views.py:23]: Pivotal: US103
https://www.pivotaltracker.com/story/show/110152938

```
Date of birth must be in the format DD/MM/YYYY
Date of birth cannot be a future date
```

File: application/templates/enterdob.html (hunk 1 above)
Comment (endamccormack) [inline application/templates/enterdob.html]: All this shouldn't be here

File: application/templates/enterdob.html [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+        <div class="form-group form-group-year">
+          <label for="dob-year">Year</label>
+          <input class="form-control" id="dob-year" name="dob-year" type="number" pattern="[0-9]*" min="0" max="2014">
+        </div>
+      </div>
+
+    </fieldset>
+
+    <div>&nbsp;</div>
```
Comment (endamccormack) [inline application/templates/enterdob.html]: Think this was accidentally left in

File: application/templates/enterdob.html [hunk 4]
```diff
@@ -0,0 +1,53 @@
+{% extends "base/lr_base_layout.html" %}
+
+{% block content %}
+  <div class="main" id="content">
+    {{ super() }}
+    <div>&nbsp;</div>
```
Comment (endamccormack) [inline application/templates/enterdob.html:6]: think this was accidentally left in

**Comment by endamccormack** [comment]
As discussed, please take note of the tech debt in the tech debt story, with this good to merge :+1:

File: application/deed/searchdeed/views.py (hunk 2 above)
Comment (Phemus) [inline application/deed/searchdeed/views.py:23]: A later email from Ian dropped this as an acceptance criteria for this story

File: application/templates/enterdob.html (hunk 4 above)
Comment (Phemus) [inline application/templates/enterdob.html:6]: This was added for formatting it will be refactored as part of a later tech debt spike

**Comment by tsangsimon** [comment]
:+1:

**Pull request merged**